docs/configuration/configuration.md

```diff
@@ -70,6 +70,10 @@ global:
 
   # How frequently to evaluate rules.
   [ evaluation_interval: <duration> | default = 1m ]
+                        
+  # Initial delay before starting the evaluation of rules to ensure the underlying metrics have been received.
```

Comment:
Same here.

Reply:
Done in https://github.com/prometheus/prometheus/pull/14061/commits/f9bfa81e3393b26165a4ce521ca59d5e7652ed5f